File: public/examples/integrations/mcp-servers/clickhouse/discover_databases_example_call_tool.py
from arcadepy import Arcade

client = Arcade()  # Automatically finds the `ARCADE_API_KEY` env variable

TOOL_NAME = "Clickhouse.DiscoverSchemas"

user_id = "{arcade_user_id}"

tool_input = {}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
    user_id=user_id,
)

print(response.output.value)

File: public/examples/integrations/mcp-servers/mongodb/discover_databases_example_call_tool.js
import { Arcade } from "arcadejs";

const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable

const TOOL_NAME = "MongoDB.DiscoverDatabases";

const userId = "{arcade_user_id}";

const toolInput = {};

const response = await client.tools.execute({
  toolName: TOOL_NAME,
  input: toolInput,
  userId: userId,
});

console.log(response.output.value);
